### PR TITLE
Turn the WG into a maintenance WG

### PR DESCRIPTION
**Pull request by pchampin**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rch-wg-charter/pull/110.html" title="Last updated on Aug 13, 2024, 10:07 AM UTC (da2fe68)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rch-wg-charter/110/2d169d7...da2fe68.html" title="Last updated on Aug 13, 2024, 10:07 AM UTC (da2fe68)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
- Having been on the receiving end of such comments no later than yesterday: you should sync the charter text with the latest charter template, otherwise this will be rejected. The new charter draft includes a bunch of small changes (e.g., all pointers to the process document are now different, there is a reference to the TAG principles, etc...), which is a pain in the back side to sync with, but there you go (the comment I got is that it is better to start with the template rather than the old charter text...)
- In particular, the charter should now begin with a separate section on "motivation and background", which should probably absorb most of what is in the scope section now. The scope is supposed to be very short now, some sort of legal text...
- I am not sure that the mission statement should stay as is. The mission of the WG is to "maintain" and not to "define". 
- Similarly, the scope should say that we maintain and not do anything more new development. This is not the way it reads right now... 
- It is probably a good idea to add a note that the WG is authorized to do class 4 changes on the spec.

**Comment by msporny** [comment]
Supportive of the charter, with @iherman's changes applied.

**Review by gkellogg — APPROVED** [review]

**Review by dlongley — APPROVED** [review]
+1 with @iherman's changes applied.

**Review by philarcher — APPROVED** [review]
LGTM, thanks PA

**Comment by pchampin** [comment]
All of @iherman's suggestions should now be addressed.

File: index.html
Comment (iherman) [inline index.html:94]:
```suggestion
        The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a> is to maintain the specifications previously published by this group. That work included a standard to uniquely and deterministically calculate a hash of RDF Datasets for use cases such as <a href="explainer.html#detectingChanges">Detecting changes in Datasets</a>, and was based  on an <a href="explainer.html#canonicalization">RDF Dataset Canonicalization</a> algorithm.
```

(Just to avoid any misunderstandings as for what the scope of ***this*** WG is...)

**Review by iherman — APPROVED** [review]
Left one comment; approving once that is taken in consideration (it is just an editorial change).

File: index.html
Comment (iherman) [reply index.html:94]: I just realize that the same sentence appears in the mission statement, too. These two should be in sync, I presume...

**Review by gkellogg — CHANGES_REQUESTED** [review]
I think the RDF-DEV CG should be replaced with the RDF-star WG for coordination. I wouldn’t be adverse to listing both, but the RDF-DEV CG isn’t working on RDF-star at this point.

**Review by dlongley — APPROVED** [review]

**Review by philarcher — APPROVED** [review]
I've committed a couple of changes to the scope section to refer to the published explainer doc (https://www.w3.org/TR/rch-explainer/) rather than the local copy which seems broken anyway. Ignore that commit if you disagree.

File: index.html
Comment (yamdan) [inline index.html:164]:
```suggestion
          There are a variety of use cases that depend on the ability to calculate a unique and deterministic hash value of RDF Datasets, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. See the <a href="https://www.w3.org/TR/rch-explainer/#usage">use cases</a> for more examples. These use cases require a standard way to process the underlying graphs contained in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```
Similar to @philarcher 's suggestion.

File: index.html
Comment (yamdan) [inline index.html:94]:
```suggestion
        The <strong>mission</strong> of the <a href="https://www.w3.org/groups/wg/rch">RDF Dataset Canonicalization and Hash Working Group</a> is to maintain the specifications previously published by this group. That work included a standard to uniquely and deterministically calculate a hash of RDF Datasets for use cases such as <a href="https://www.w3.org/TR/rch-explainer/#detectingChanges">Detecting changes in Datasets</a>, and was based  on an <a href="https://www.w3.org/TR/rch-explainer/#canonicalization">RDF Dataset Canonicalization</a> algorithm.
```
Similar to @philarcher 's suggestion.

**Review by yamdan — APPROVED** [review]
Thank you @pchampin. I approve it with a few editorial suggestions similar to those made by @philarcher.

**Comment by pchampin** [comment]
@peacekeeper I need you to approve the PR before I can merge it

**Review by gkellogg — APPROVED** [review]

File: index.html
Comment (TallTed) [inline index.html:164]:
```suggestion
          Various use cases depend on the ability to calculate a unique and deterministic hash value of RDF Datasets, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, and the consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. See the <a href="https://www.w3.org/TR/rch-explainer/#usage">use cases</a> for more examples. These use cases require the existence of a standard way to process the underlying graphs contained in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```

File: index.html
Comment (TallTed) [inline index.html:171]:
```suggestion
          The scope of this Working Group is to maintain the specifications published under the previous charter. That work included a standard to <a href="https://www.w3.org/TR/rch-explainer/#canonicalization">canonicalize</a> and <a href="https://www.w3.org/TR/rch-explainer/#goals">cryptographically hash</a> an <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Dataset</a>, and was based on the definition of a standard <a href="https://www.w3.org/TR/rch-explainer/#canonicalization">canonicalization algorithm</a>. (See the separate <a href="https://www.w3.org/TR/rch-explainer/">explainer document</a> for more detailed technical background and for the terminology used in this context.)
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor, editorial

**Comment by pchampin** [comment]
I still am unable to merge. Probably because @TallTed requested changes, which I just accepted. @TallTed can you now approve the PR?

**Review by TallTed — APPROVED** [review]

**Pull request merged**
